modules/reference/hardware/flake-module.nix

```diff
@@ -45,6 +45,14 @@
       }
     ];
 
+    hardware-intel-laptop.imports = [
+      inputs.self.nixosModules.hardware-x86_64-workstation
+      {
+        ghaf.hardware.definition = import ./intel-laptop/intel-laptop.nix;
+        ghaf.hardware.usb.vhotplug.autoDetectPci = true;
+      }
+    ];
```

Comment:
great. How much of our current list of HW can we support with this configuration as the base? e.g. could the x1-gen11 carbon inherit this and add only 1 feature. or the x1-gen13 uses it entirely?